src/game/frontend/items/FloatCommandItem.cpp

```diff
@@ -6,7 +6,8 @@
 
 namespace YimMenu
 {
-	FloatCommandItem::FloatCommandItem(joaat_t id, std::optional<std::string> label_override) :
+	FloatCommandItem::FloatCommandItem(joaat_t id, std::optional<std::string> label_override, bool use_slider) :
+		m_useSlider(use_slider),
```

Comment:
The reason this looks messed up is because I am using tabs as spacing exclusively, while the lines below seem to have a combination of tabs and spaces.